config/config.go

```diff
@@ -496,6 +496,9 @@ func CheckConfig() {
 		}
 	}
 	if !ncutils.FileExists(GetNetclientPath() + "netmaker.conf") {
+		if err := os.MkdirAll(GetNetclientPath(), os.ModePerm); err != nil {
+			logger.Log(0, "failed to create /etc/netclient", err.Error())
+		}
```

Comment:
is directory creation happening else where?

Reply:
yes, but calling MkdirAll() from multiple places is not an issue.

// MkdirAll creates a directory named path,
// along with any necessary parents, and returns nil,
// or else returns an error.
// The permission bits perm (before umask) are used for all
// directories that MkdirAll creates.
// If path is already a directory, MkdirAll does nothing
// and returns nil.